### PR TITLE
feat: add vault yield event txs, fix seed for vault snapshot

### PR DESCRIPTION
**Pull request by NamelsKing**

### CODE REVIEW

File: src/vaults/__snapshots__/harvests.utils.spec.ts.snap [hunk 1]
```diff
@@ -237,6 +255,7 @@ Array [
     "grossApr": 10.650635139472206,
     "timestamp": 1651437281000,
     "token": "0x62B9c7356A2Dc64a1969e19C23e4f579F9810Aa7",
+    "tx": undefined,
```
Comment (axejintao) [inline src/vaults/__snapshots__/harvests.utils.spec.ts.snap:258]: should these be undefined?

Comment (NamelsKing) [reply src/vaults/__snapshots__/harvests.utils.spec.ts.snap:258]: not, but would be easier to fix, after mocks.pkg upd, forgot to do that

Comment (NamelsKing) [reply src/vaults/__snapshots__/harvests.utils.spec.ts.snap:258]: can do as a follow up

**Review by axejintao — APPROVED** [review]
sgtm, will look for follow up

**Pull request merged**